Project/app/src/main/java/org/ionproject/android/class_section/ClassSectionFragment.kt

```diff
@@ -0,0 +1,55 @@
+package org.ionproject.android
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.ViewModelProviders
+import kotlinx.android.synthetic.main.fragment_class_section.*
+import org.ionproject.android.class_section.ClassSectionViewModel
+import org.ionproject.android.class_section.ClassSectionViewModelProvider
+
+/**
```

Comment:
This is temporary, right? It does not document anything. If there isn´t much to say, then we should focus on our domain: for example, the class implements a fragment used to display a class section. If it's only used in some contexts, then state it too.

Reply:
Yes, it's temporary